src/css/components/_performance-audit.scss

```
@@ -0,0 +1,698 @@
+	&__label {
+		padding: 12px;
+		color: var(--wppo-text-muted);
+		font-weight: 500;
+		width: 45%;
+	}
+
+	&__value {
+		padding: 12px;
+		color: var(--wppo-text-main);
+		font-weight: 500;
+		word-break: break-word;
```

Stylelint (17.7.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)